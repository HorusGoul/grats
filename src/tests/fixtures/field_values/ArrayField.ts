/** @gqlType */
export default class Query {
  /** @gqlField */
  hello(): Array<string> {
    return ["Hello world!"];
  }
}
