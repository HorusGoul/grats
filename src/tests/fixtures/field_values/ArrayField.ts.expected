-----------------
INPUT
----------------- 
/** @gqlType */
export default class SomeType {
  /** @gqlField */
  hello(): Array<string> {
    return ["Hello world!"];
  }
}

-----------------
OUTPUT
-----------------
directive @asyncIterable on FIELD_DEFINITION

directive @methodName(name: String!) on FIELD_DEFINITION

directive @exported(jsModulePath: String!, tsModulePath: String!, functionName: String!) on FIELD_DEFINITION

type SomeType {
  hello: [String!]
}